README.md

```diff
@@ -107,3 +107,32 @@ Meeting Agenda is maintained [here](https://github.com/redhat-developer/service-
 Please file bug reports on [Github](https://github.com/redhat-developer/service-binding-operator/issues/new). For any other questions, reach out on `service-binding-support@redhat.com`.
 
 
+## Roadmap
+
+
+**Key**
+
+⚪️ Concept validated
+
+:white_check_mark: Works, has tests and at least one documented scenario
+
+
+
+### Maturity matrix
+
+| Feature                                        | Alpha                | Beta | GA |
+| -----------------------------------------------| -------------------- | ---- | -- |
+| Binding with Kubernetes resources              |  :white_check_mark:  |      |    |
+| Binding with CRD-backed services               |  :white_check_mark:  |      |    |
+| Binding with OLM CRD-backed services           |  :white_check_mark:  |      |    |
+| Binding with secrets/configmaps                |  :white_check_mark:  |      |    |
+| Binding of PodSpec-based workloads             |  :white_check_mark:  |      |    |
+| Binding of non-PodSpec-based workloads         |  ⚪️    |      |    |
```

Comment:
most of the time we use the term `applications`, now here we have `workloads`. Let's be consistent.

Reply:
Reason I used `workloads` is because `podSpec`-based applications isn't a commonly used term. Usually, `podSpec`-based workloads is the term that's friendlier to kubernetes users.

Reply:
do we really need to emphasize here `podSpec` at all? Could we simply not list all k8s resources containing podSpec?

Reply:
We do, to reflect that CRs with a podSpec are perfectly legal in the eyes of Service Binding.